utils.py

```
@@ -44,3 +44,35 @@ def plot_network_output(real_A, fake_B, real_B, fake_A, iteration):
 def create_image(im):
     # scale the pixel values from [-1,1] to [0,1]
     return (im + 1) / 2
+
+def create_traintest(inputdir,outputdir='./data',AB='A',train_frac = 0.75):
+    '''
+    inputs:
+        inputdir - directory with images of one class (ex: sunny_beach or cloudy_beach)
+        outputdir - directory to save the new train/test directories to. Default is cwd/data
+        AB - for GANS, rename to either class 'A' or class 'B'
+        train_frac - fraction of images in train vs test set
+
+    outputs:
+        New directories in current working directory of testA, testB, trainA, trainB
+    '''
+    #read in list of all files in inputdir and shuffle
+    all_files = os.listdir(inputdir)
+    np.random.shuffle(np.array(all_files))
```

Good idea. Do you think it should default to random if no seed passed, or just a default seed that is also repeatable?